plugins/language_tal.lua

```diff
@@ -1,29 +1,28 @@
 --mod-version:3
-local syntax = require 'core.syntax'
 
 local label, sublabel = "function", "keyword2"
 
-syntax.add {
+require 'core.syntax'.add {
```

Comment:
Why?

Reply:
core.syntax is only used that one time

Reply:
Yeah, like basically every other language syntax plugin. No reason to make this less legible.

Reply:
this is arguably just as readable, but alright then